sentry/src/main/java/io/sentry/Scope.java

```diff
@@ -337,9 +338,10 @@ public void clear() {
    *
    * @return the tags map
    */
-  @NotNull
-  Map<String, String> getTags() {
-    return tags;
+  @ApiStatus.Internal
+  @SuppressWarnings("NullAway") // tags are never null
+  public @NotNull Map<String, String> getTags() {
```

Comment:
```suggestion
  public @NotNull Map<@NotNull String, @NotNull String> getTags() {
```

does it work without `@SuppressWarnings("NullAway")`?

Reply:
`Collections.newConcurrentHashMap` returns a nullable object, this is why this one is marked with `SuppressWarnings`, as the `tags` are never null.

Reply:
makes sense, thanks